tests/infra/ccf.py

```diff
@@ -472,28 +483,35 @@ def wait_for_sealed_secrets_at_version(self, version, timeout=5):
 
 
 @contextmanager
-def network(hosts, build_directory, dbg_nodes=[], perf_nodes=[], pdb=False, txs=None):
+def network(
+    hosts, binary_directory=".", dbg_nodes=[], perf_nodes=[], pdb=False, txs=None
+):
     """
     Context manager for Network class.
     :param hosts: a list of hostnames (localhost or remote hostnames)
-    :param build_directory: the build directory
+    :param binary_directory: the directory where CCF's binaries are located
```

Comment:
Is there other stuff out of the install directory we could want that would make it worth passing that and constructing install_dir/bin instead (as far as I can tell, no, this is a genuine question)?

Reply:
I don't think there's anything else we want in the install directory. Maybe the Python infra could be found without `PYTHONPATH` twiddling, but we're already _in_ the Python infra so probably not.

On a related point, perhaps these binaries should actually be in `build/bin` locally, to mirror the `install_dir/bin` they end up in, and to make sure we're not just getting lucky finding something in the build-dir root.